### PR TITLE
Improve Serilog log injection doc

### PR DESCRIPTION
**Pull request by pellared**
## Why

The current doc is not very straightforward.

## What

- Improve Serilog log injection doc
- Fix `dev/envvars.sh` to make it working on Windows 

## Tests

I was changing the Serilog example and playing with different configurations. I was testing it also against the Console sink and I was trying different output templates.

## Next PRs

Improve description for other logging libraries.

### CODE REVIEW

**Review by Kielek — APPROVED** [review]

File: docs/correlating-traces-with-logs.md [hunk 1]
```diff
@@ -80,8 +80,13 @@ to print out all contextual properties.
 
 Alternatively, for more fine-grained control,
 you can use the trace context fields explicitly.
-However, the values must be surrounded with a quotation mark
-(e.g. `service.name=\"{service_name}\"`).
+However, the values must be surrounded with a quotation mark.
+For instance, you can use following output template,
+which also transforms the field names:
```
Comment (theletterf) [inline docs/correlating-traces-with-logs.md]:
```suggestion
When using trace context fields manually. wrap values in
quotation marks. For instance, you can use the following output template,
which also transforms the field names:
```

**Review by theletterf — APPROVED** [review]

File: docs/correlating-traces-with-logs.md (hunk 1 above)
Comment (pellared) [reply docs/correlating-traces-with-logs.md]: The comment was done on outdated version 😉

File: docs/correlating-traces-with-logs.md [hunk 2]
```diff
@@ -1,30 +1,33 @@
...
-corresponding events. If your application uses a supported logger, enable trace
-injection to automatically include trace context in your application's logs.
+If your application uses a supported logger,
+you can configure log correlation to
+include trace context in your application's logs.
 
-To inject traces in logs, enable log correlation by setting the environment variable
+To inject trace context fields in logs,
+enable log correlation by setting the environment variable
 `SIGNALFX_LOGS_INJECTION=true` before running your instrumented application.
```
Comment (theletterf) [inline docs/correlating-traces-with-logs.md:9]:
```suggestion
enable log correlation by setting the `SIGNALFX_LOGS_INJECTION` 
environment variable to `true` before running your instrumented application.
```

File: docs/correlating-traces-with-logs.md [hunk 3]
```diff
@@ -1,30 +1,33 @@
...
 
-If your logger uses JSON, the tracing library automatically handles trace ID
-injection.
+If your logger uses JSON logging format,
+then SignalFx Instrumentation for .NET automatically adds
+span context to the logs.
```
Comment (theletterf) [inline docs/correlating-traces-with-logs.md:13]:
```suggestion
If your logger uses JSON as the logging format,
the SignalFx Instrumentation for .NET automatically adds
span context to the logs.
```

File: docs/correlating-traces-with-logs.md [hunk 4]
```diff
@@ -1,30 +1,33 @@
...
 
-If your logger uses a raw format, manually configure your logger to include
-the trace ID and span ID.
+If your logger uses a different format,
+you may have to manually configure your
+logger to include the trace context fields.
```
Comment (theletterf) [inline docs/correlating-traces-with-logs.md:17]:
```suggestion
If your logger uses a different format,
you might have to manually configure your
logger to include the trace context fields.
```

File: docs/correlating-traces-with-logs.md [hunk 5]
```diff
@@ -44,18 +49,47 @@ Available layouts:
...
+- [NLog 4.6.x](../tracer/samples/AutomaticTraceIdInjection/NLog45Example).
+
 ### Serilog
 
 - Versions: 1.4.0 ≤ 2.*.*
 
-Available layouts:
+Regardless of the output layout, your `LoggerConfiguration` must be
+enriched from the LogContext to extract the trace context
+that is automatically injected.
```
Comment (theletterf) [inline docs/correlating-traces-with-logs.md:64]:
```suggestion
that is automatically injected. For example:
```

File: docs/correlating-traces-with-logs.md [hunk 6]
```diff
@@ -44,18 +49,47 @@ Available layouts:
...
-\* Log transformation rules must be configured as Serilog doesn't support
+When using the output template you can either use `{Properties}`
+to print out all contextual properties.
+
+Alternatively, for more fine-grained control,
+you can use the trace context fields explicitly.
+The values MUST be wrapped in quotation marks.
+For instance, you can use following output template,
+which also transforms the field name
+(the log transformation step would not be required):
```
Comment (theletterf) [inline docs/correlating-traces-with-logs.md:86]:
```suggestion
When using the output template you can either use `{Properties}`
to print out all contextual properties or add context fields manually.

When adding context fields manually, values must be wrapped in 
quotation marks. For instance, you can use following output template,
which also transforms the field name (the log transformation step would not be required):
```

**Review by theletterf — CHANGES_REQUESTED** [review]

**Review by theletterf — APPROVED** [review]

**Pull request merged**